recipes/itolparser/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "itolparser" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b4271b168e24859ea806388e934c6bb1eda18ee70ab6c8ae236dbb2ca17c9ebf
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - colorbrewer
+    - numpy
+    - pandas
+    - pip
+    - python >=3.6
+  run:
+    - colorbrewer
+    - numpy
+    - pandas
+    - python
+
+test:
+  imports:
+    - bin
```

Comment:
Not really related to the conda recipe, but itolparser would be a better name for this import, could you rename the directory in your repo?

You can also create an entry point for your itolparser script: https://packaging.python.org/guides/distributing-packages-using-setuptools/#entry-points